src/components/Common/UserAutocompleteFormField.tsx

```diff
@@ -80,8 +80,11 @@ export default function UserAutocomplete(props: UserSearchProps) {
         )}
         optionLabel={formatName}
         optionIcon={userOnlineDot}
+        optionImage={(option: UserBareMinimum) =>
+          option.read_profile_picture_url || ""
```

Comment:
why should we cast it to an empty string 🤔, we should be able to deal with an option right 🤔

Reply:
![Screenshot from 2024-10-24 11-31-36](https://github.com/user-attachments/assets/03064099-5ea5-41b0-84fc-d9751d1444a2)
There is an error

Reply:
you can make the `optionImage` accept undefined right?